browser/extensions/api/settings_private/brave_prefs_util.cc

```diff
@@ -67,6 +68,10 @@ const PrefsUtil::TypedPrefMap& BravePrefsUtil::GetWhitelistedKeys() {
   // Hangouts pref
   (*s_brave_whitelist)[kHangoutsEnabled] =
       settings_api::PrefType::PREF_TYPE_BOOLEAN;
+  (*s_brave_whitelist)[rewards_data::prefs::kRewardsAutoContributeSites] =
```

Comment:
why are these here?

Reply:
These are to whitelist the prefs to be used for polymer `settings-checkbox`